environment-approvals.yml

```diff
@@ -80,6 +80,7 @@ prod:
   - repo: https://github.com/hmcts/ia-shared-infrastructure.git
   - repo: https://github.com/hmcts/ia-aip-frontend.git
   - repo: https://github.com/hmcts/ia-timed-event-service.git
+  - repo: https://github.com/hmcts/ia-home-office-integration-api.git
```

Comment:
can you link to your approval please?

Reply:
Please find below link to documentation:
https://tools.hmcts.net/confluence/display/RSTR/Immigration+and+Asylum+-+Rel+4.0.0+-+Home+Office+Integration+API+1